libs/utils/deps/login.js

```diff
@@ -0,0 +1,25 @@
+/* global msal */
```

Comment:
Not a fan of naming the folder "deps".  login.js and msal.js should be moved into something like libs/utils/sharepoint/

state.js seems like it's specific to version-history, so should be moved into the block folder

Reply:
I think using vanilla `/libs/utils` might be too generic. These are not really in the same category as utils.js or some of the more visitor-facing stuff we have in there. All of this is author-facing.

**How about:**
```
/libs/deps/sharepoint/login.js
/libs/deps/sharepoint/msal.js
```

My only argument against this would be that typically we put stuff we don't maintain into deps, and we _would_ be maintaining these files.

Reply:
@auniverseaway state.js is used by login.js and masl.js for the auth token and site configs, that is why i kept it under the same folder. The same files will be used for locui login as well.